src/artm/regularizer/net_plsa_phi.h

```diff
@@ -0,0 +1,68 @@
+/* Copyright 2017, Additive Regularization of Topic Models.
+
+   Author: Murat Apishev (great-mel@yandex.ru)
+
+   This class proceeds Phi matrix regularization due to NetPLSA strategy.
+   The formula of M-step is
+   
+   p_ut \propto n_ut + tau * \sum_{v \in U} w_uv n_t^2
+                             ([p_vt / (|D_u| * |D_v|)] - [p_ut / (|D_u|^2)]),
+   
+   where U is the set of tokens of specific vertex modality of documents graph,
+   w_uv is a weight of edge between vertices u and v,
+   n_t is a normalized n_wt for tokens of U class_id,
+   D is an array of weights of vertices, len(D) == |U|
```

Comment:
"n_t is a normalized n_wt for tokens of U class_id" == n_t = \sum_{v \in U} n_vt